File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 12/28/2025, 4:53:38 PM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        s=0
        for i in range(1,n+1):
            s+=i
        return s-sum(nums)

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 12/28/2025, 4:53:35 PM
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int s=n*(n+1)/2;
        for(int i:nums) s-=i;
        return s;
    }
}